bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/ErrorValue.java

```diff
@@ -302,9 +302,20 @@ public String getPrintableStackTrace() {
         for (int i = 1; i < stackTrace.length; i++) {
             printStackElement(sb, stackTrace[i], "\n\t   ");
         }
+        addErrorCauseWithLocation(sb, this.cause);
```

Comment:
Should we fix this in place we print the stacktrace. getPrintableStackTrace is for given error stack.